ang/my-activities.js

```diff
@@ -0,0 +1,3 @@
+(function (angular, $, _) {
```

Comment:
I guess this is some kind of standard, but I don't see in use the second and third parameter. Can't be removed on that case?